.travis.yml

```diff
@@ -5,15 +5,7 @@ addons:
       - aspell
       - aspell-en
 before_install:
-  - 'cpanm --notest App::cpm'
```

Comment:
Why revert this change?  `cpm` installs in parallel.  This will make the Travis builds slower.

Reply:
Hi @oalders, thank you for merging the change. I removed this line (and lines 10-13) to revert back to the travis-perl defaults. Travis was failing to build on all versions. See https://travis-ci.org/davel/OrePAN2/jobs/620791401 which attempts to build f32a01504832ee72c8077efd6b294dd67830ddaf, which was your most recent revision.

It used to work, https://travis-ci.org/tokuhirom/OrePAN2/builds/450006637 builds the exact same revision. I suspect something external.

Reply:
Hi @davel.  Thanks for noticing that.  See https://github.com/travis-perl/helpers/issues/71  This needs to be fixed in the helpers or I think just using `dist: trusty` should get the older Perls working again.